msg/js/ba.js

```diff
@@ -316,17 +324,17 @@ Blockly.Msg.TEXT_CREATE_JOIN_TITLE_JOIN = "ҡушылығыҙ";
 Blockly.Msg.TEXT_CREATE_JOIN_TOOLTIP = "Add, remove, or reorder sections to reconfigure this text block.";  // untranslated
 Blockly.Msg.TEXT_GET_SUBSTRING_END_FROM_END = "to letter # from end";  // untranslated
 Blockly.Msg.TEXT_GET_SUBSTRING_END_FROM_START = "# хатҡа";
-Blockly.Msg.TEXT_GET_SUBSTRING_END_LAST = "һуңғы хәрефкә тиклем";
+Blockly.Msg.TEXT_GET_SUBSTRING_END_LAST = "to last letter";  // untranslated
```

Comment:
You have a lot of regressions here.  Better just revert everything in the msg/js directory.